.expeditor/verify.pipeline.yml

```diff
@@ -1,10 +1,8 @@
 steps:
   - label: ":linux: :ruby:"
     commands:
-      - ruby --version
-      - bundler --version
-      - bundle install
-      - bundle exec rake
+      - chmod +x /workdir/.expeditor/buildkite/verify.sh
```

Comment:
You can chmod +x on your local machine then check in the verify.sh - the permissions will be updated in github.

Reply:
For some reason it did not work when I attempted to do it - but I will try it once more

Reply:
Huh it really should work.

Reply:
trying again now :)